.github/ROADMAP.md

```diff
@@ -1,125 +1,28 @@
-
-What tasks will be picked?  This will be decided in the following months.
-
-## Overview
-- **KerasCV is not a repository of blackbox end-to-end solutions (like TF Hub or ModelGarden).**
-
-    For the time being, it is focused on modular and reusable building blocks. We expect models in
-    ModelGarden or TF Hub to eventually be built on top of the building blocks provided by KerasCV.
-
-    In the process of developing these building blocks, we will by necessity implement end-to-end
-    workflows, but they're intended purely for demonstration and grounding purposes, they're not
-    our main deliverable.
-
-
-- **KerasCV is not a repository of low-level image-processing ops, like tf.vision.**
-
-    KerasCV is fundamentally an extension of the Keras API: it hosts Keras objects, like layers,
-    metrics, or callbacks. Low-level C++ ops should go elsewhere.
-
-
-- **KerasCV is a production grade library.**
-
-    Researchers may use it, but we do not consider researchers to be our target audience. Our target
-    audience is applied CV engineers with experimentation and production needs. KerasCV should make
-    it possible to quickly reimplement industry-strength versions of the latest generation of
-    architectures produced by researchers, but we don't expect the research effort itself to be built
-    on top of KerasCV. This enables us to focus on usability and API standardization, and produce
-    objects that have a longer lifespan than the average research project.
-
-# Community contribution guideline
-We would like to leverage/outsource the Keras community not only for bug reporting or fixes,
-but also for active development for feature delivery. To achieve this, we will have the predefined
-process for how to contribute to this repository.
-
-The ideal workflow will be:
-
-- Keras defines the API interface for a certain task, eg a layer, model, dataset, metrics, etc.
-- Post it to Github as a feature request.
-- Have a hotlist/tag for those items and advocate them to the community.
-- Let community members claim the task they would like to work on.
-
-User requested features will be carefully evaluated by keras team member before acceptance.
-Based on the existing experience from tf-addons, even with a guideline of paper ref count > 50,
-the image/cv related contributions are still quite scattered. We will make sure the contributions
-are coherent with the tasks/themes we have in the repository, or widely used for CV tasks.
+The team will release 2 quarters of roadmap in advance so contributors will know
+what we are working on and be better aligned when creating PRs.  
+As an exception, widely used backbones are always welcome. Contributors can search for `contribution-welcome` label on github.
+The team will release one minor version upgrade each quarter, or whenever a new task is officially supported.
+
+2023 Q1:
+- 3D Object Detection task, the team focuses on providing a point cloud based model (specific model TBD)
```

Comment:
@tanzhenyu any chance we could see nerf-based model, [like](https://github.com/nerfstudio-project/nerfstudio). 😶‍🌫️ 
(asking from curiosity)

Reply:
/cc @taya

Reply:
> @tanzhenyu any chance we could see nerf-based model, [like](https://github.com/nerfstudio-project/nerfstudio). 😶‍🌫️ (asking from curiosity)

This would apply for the 5) rule which is to create a task in the contribution guide, a complete new task which is not within our immediate roadmap. We would need external contributors to self-initiate this with sponsors (from KerasCV authors, or someone expertise in this sub-field), going over design review process (with their names on RFC), and end to end implementation